src/io.rs

```diff
@@ -11,6 +17,33 @@ use log::{debug, info, warn};
 
 use crate::juno_manager::ManagerError;
 
+pub fn path_for_overall_report() -> PathBuf {
+    PathBuf::from("results/class_hashes/overall_report.json".to_string())
+}
+
+pub fn successful_comparison_glob() -> String {
+    "results/comparison-*.json".to_string()
+}
```

Comment:
Re-locate this functions to be placed next to their similars (e.g. next to `succesful_comparison_path`

Reply:
Done.